core/trino-server/src/main/provisio/trino.xml

```diff
@@ -291,6 +291,12 @@
         </artifact>
```

Comment:
Following https://trino.io/development/process#pull-request-review, the commit history is also part of the code review. From what I understand it looks like they all needs to be squashed into a single commit - "Introduce Apache Ranger system access control plugin"

Reply:
@kokosing  - should all commits be squashed for the review? I expected squashing will be done during merge.